static/app/components/tabs/index.tsx

```diff
@@ -31,6 +31,10 @@ export interface TabsProps<T>
    * selected tab item.
    */
   defaultValue?: T;
+  /**
+   * Disable tabs from being put in the overflow menu.
+   */
+  disableOverflow?: boolean;
```

Comment:
displaying tabs in flex/grid really gets annoying if the flex isn't set to grow. needed the option to just disable overflow

annoying overflow for no reason
![image](https://github.com/user-attachments/assets/aabda4eb-8f1c-4f4d-b3c3-bda3b4b4a717)